spec/unit/resource/api800/c7000/storage_system_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API800::C7000::StorageSystem do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API600::C7000::VolumeTemplate' do
```

Comment:
it should be StorageSystem in place of VolumeTemplate

Reply:
done